kk3528.md

```diff
@@ -0,0 +1,14 @@
+# Open Source Development
+
+Name: Kutay Karakaş
+
+UNI: kk3528
+
+Github: [link](https://github.com/kkarakas)
+
+GitHub README: [link](https://github.com/kkarakas/kkarakas/blob/main/README.md)
+## Project Proposal
+A terminal tool that downloads all of Prof. Paine's slide in pdf format with one command.  
```

Comment:
lol, fair enough

Reply:
My grade for hw0 was lowered due to not having proposal. Would you like me to detail it more for hw0?